lenskit/algorithms/als.py

```diff
@@ -584,6 +583,19 @@ def fit(self, ratings, **kwargs):
                      np.linalg.norm(self.user_features_, 'fro'),
                      np.linalg.norm(self.item_features_, 'fro'))
 
+        # unpack the regularization
+        if isinstance(self.reg, tuple):
+            ureg, ireg = self.reg
+        else:
+            ureg = ireg = self.reg
+
+        # compute OtOr and save it on the model
+        nf = self.item_features_.shape[1]
+        regmat = np.identity(nf) * ureg
+        Ot = self.item_features_.T
+        OtO = Ot @ self.item_features_
+        self.OtOr = OtO + regmat
```

Comment:
Learned parameters should end in underscores - `self.OtOr_`

Reply:
Done